airflow/operators/python_operator.py

```diff
@@ -86,6 +87,38 @@ def __init__(
         if templates_exts:
             self.template_ext = templates_exts
 
+    def __deepcopy__(self, memo):
+        """
+        Hack sorting double chained task lists by task_id to avoid hitting
+        max_depth on deepcopy operations.
+        """
+        sys.setrecursionlimit(5000)  # TODO fix this in a better way
```

Comment:
This is probably already set from the caller at that point, no need to copy paste

Reply:
thanks. will remove this.